tslint.json

```diff
@@ -1,37 +1,21 @@
 {
-	"rules": {
+  "extends": [
+    "tslint-config-prettier"
```

Comment:
I ended up having to do what you were suggesting on #56 to make sure tslint and prettier didn’t conflict, @mcMickJuice.